pages/adult-form.js

```diff
@@ -0,0 +1,20 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+
+const Adult = () => {
+  const router = useRouter();
+  return (
+    <div>
+      <Head>
+        <title>Adult</title>
+        <link rel="icon" href="/favicon.ico" />
```

Comment:
Not a big deal but I would probably move the `favicon` in the `_document` page, my fault that I copy and pasted the index from the `create-next-app` 😅